src/main/java/se/munchbox/userPref/UserPreference.java

```diff
@@ -0,0 +1,74 @@
+package se.munchbox.userPref;
+
+
+import se.munchbox.user.User;
+
+import javax.persistence.*;
+
+@Entity
+public class UserPreference {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(columnDefinition = "TEXT")
+    private String userPrefBody;
+    @Column(columnDefinition = "TEXT")
+    private String userPrefTitle;
+
+    @OneToOne
+    private User user;
```

Comment:
Ok, so as of now the User has 1 to 1 with UserPreference, and User preference has a string body and a string title. This is great for now, possibly we will only need a title (to be discussed later)